src/status_im/network/core.cljs

```diff
@@ -81,26 +93,34 @@
    (when handler
      (handler data cofx))))
 
+;TODO(tbenr) IMHO there is some confision between network-id (chain-id) and network-id (id in db)
```

Comment:
Could you create an issue to track that?

Reply:
I created #7829

Reply:
Thanks! Maybe we can remove the todo then to reduce duplication?